.github/workflows/render-and-check.yaml

```diff
@@ -0,0 +1,29 @@
+name: Render & check markdown
+
+on:
+  push:
+  pull_request:
+    branches: [master]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - run: yarn install
+
+      - name: "Render docs"
+        run: yarn render:cns-contracts
```

Comment:
As far as I remember, we were a bit paranoid adding it as a CI step and agreed to keep it just as a script in package.json for now. So I'd revert the changes to the workflow files. But if anyone wants to suggest otherwise, please feel free to do so. Anyway, it's good to see that it's pretty easy to set up, so we can consider this in future.

Reply:
Yes, I added this to consider such option. Docs can be updated automatically if we'll want so.

Reply:
Reverted for now.